src/secret.rs

```diff
@@ -256,7 +256,7 @@ impl<'a> From<&'a SecretKey> for ExpandedSecretKey {
     ///
     /// # Examples
     ///
-    /// ```
+    /// ```ignore
```

Comment:
Why are these ignored now?

Reply:
Because they are not part of public API that you could use, and leaving them runnable will fail the doc test
(I didn't remove these rustdoc cuz they seem useful just as example for internal usage)